stable/redis-ha/templates/redis-master-appliance-vpc.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-master-appliance-vpc
+  annotations:
+{{ toYaml .Values.appliance.annotations | indent 4 }}
+spec:
+  ports:
+  - port: 6379
+    protocol: TCP
+    targetPort: 6379
+  selector:
+    app: "redis-ha"
+    redis-node: "true"
+    redis-role: "master"
+    release: "{{ .Release.Name }}"
+{{- if eq .Values.appliance.vpcMode true }}
```

Comment:
Would prefer this just value in the values file that is defaulted to ClusterIP. For example:
https://github.com/kubernetes/charts/blob/master/stable/prometheus/values.yaml#L156